configure.ac

```diff
@@ -3231,6 +3231,10 @@ if test "X$SUBFILING_VFD" = "Xyes"; then
       AC_MSG_ERROR([Subfiling VFD requires atomic operations support. C11 stdatomic.h NOT available.])
     fi
 
+# Checks for libraries.
+    AC_CHECK_LIB([rt], [shm_open])
```

Comment:
We already check for rt in the context of clock_gettime (line 2090). You could probably replace the clock_gettime check (posix4 is archaic, IIRC) with this one. Also, in VFD SWMR I have to always link to pthreads on MacOS (though I'd have to double check to be sure).

Reply:
I saw that, but for some reason, the testing on Jelly showed that it was not adding it.

Reply:
It's using AC_SEARCH_LIBS, which first checks to see if the symbol is available without linking to anything new. If you change that to AC_CHECK_LIB, you'll always link. AC_SEARCH_LIBS will also check for libposix4, but that functionality was moved to libc a long time ago, making libposix4 archaic and we can ignore it.

Reply:
Or you could do two AC_SEARCH_LIBS calls, one for clock_gettime and one for shm_open. AC_SEARCH_LIBS is considered preferable to AC_CHECK_LIB, as the former won't bring in anything new unless it's necessary.